src/Blocks/math.jl

```diff
@@ -1,5 +1,5 @@
 """
-    Gain(k; name)
+    Gain(; name, k)
```

Comment:
This is a breaking change. Could the old constructor be manually added back to avoid releasing a new major version?

Reply:
Done: https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/189/files#diff-e3b4ead98757cf11374489c55daa0f847f043f0de07235f7a5e7c448fca9a38fR24